docs/deploying/deploy-smart-contracts.mdx

```diff
@@ -64,6 +64,4 @@ For production-grade applications, it's recommended to obtain your own API keys
 
 - `ETHERSCAN_API_KEY` variable in `packages/hardhat/.env` with your generated API key. You can get your key [here](https://etherscan.io/myapikey).
 
-:::tip Hint
-It's recommended to store env's for nextjs in Vercel/system env config for live apps and use .env.local for local testing.
-:::tip Hint
+:::tip Hint It's recommended to store env's for nextjs in Vercel/system env config for live apps and use .env.local for local testing. :::tip Hint
```

Comment:
```suggestion
:::tip Hint
It's recommended to store env's for nextjs in Vercel/system env config for live apps and use .env.local for local testing. 
:::tip Hint
```

I think `:::tip HInt` should be on new line otherwise it does not render properly

Reply:
Prettier fixed and code fixed! 😅